var/spack/repos/builtin/packages/py-lifelines/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyLifelines(PythonPackage):
+    """Survival analysis was originally developed and applied heavily
+    by the actuarial and medical community. Its purpose was to answer
+    *why do events occur now versus later* under uncertainty (where
+    *events* might refer to deaths, disease remission, etc.). This is
+    great for researchers who are interested in measuring lifetimes:
+    they can answer questions like *what factors might influence
+    deaths?*
+    But outside of medicine and actuarial science, there are many
+    other interesting and exciting applications of survival
+    analysis. For example:
+
+    - SaaS providers are interested in measuring subscriber lifetimes,
+      or time to some first action
+    - inventory stock out is a censoring event for true "demand" of a good.
+    - sociologists are interested in measuring political parties'
+      lifetimes, or relationships, or marriages
+    - A/B tests to determine how long it takes different groups to
+      perform an action.
+    *lifelines* is a pure Python implementation of the best parts of
+      survival analysis."""
```

Comment:
`spack info` will probably break this formatting. Do we really need such a long description? A single sentence or two would probably be sufficient.

Reply:
shortened